### ISSUE TITLE
Multiple drone test

### ISSUE THREAD

**Issue by PhilPhal** [ctu-mrs/mrs_uav_system#83]
Hi MRS,

I would like to perform a real-life test with multiple drones but can't find a template of a just_flying.sh to do this. Is there one somewhere? I could use an example ;)

Thank you in advance.

Best regards,
Philippe

**Comment by lidiaxp** [comment]
Hi, the MRS's explanation to use multiple drones are in https://ctu-mrs.github.io/docs/simulation/drone_spawner.html

I use the files in [this paste](https://drive.google.com/drive/folders/1JeDfhyCfbOXd9G_YE1Xitkr4HScl4GOp?usp=sharing) to launch multiple UAVs. You just need the 13 line on session.yml to set your launch. I will put the launch as sample too, the main change is in the line 14 to set the name of your world, and in the line 17 to set the path of your world (both in the launch file).

**Comment by DanHert** [comment]
@lidiaxp I think Phil wants to fly with real drones, not in simulation .

**Comment by PhilPhal** [comment]
Yes indeed! Thank you very much for the help, I will take a look at it.

Best regards,
Philippe
